.github/workflows/checks.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
     - name: Ensure latest install manifest
       run: |
         echo "" > dist/install.yaml
-        IMG=ghcr.io/maiqueb/kubevirt-ipam-claims:latest make build-installer
+        IMG=ghcr.io/kubevirt/ipam-controller:latest make build-installer
```

Comment:
i am not sure about it
see please
https://github.com/kubevirt/ipam-extensions/pull/31#discussion_r1654210448

Reply:
This is pushing the kubevirt-ipam-controller image from kubevirt/ipam-extensions repo, it make sense to me

We may have multiple controllers under kubevirt/ipam-extension repo, this is an image of one of them.

Reply:
https://github.com/kubevirt/ipam-extensions/actions/runs/9679520037
it fails, forbidden
to rename then as org/repo ?

```
ERROR: failed to solve: failed to push ghcr.io/kubevirt/ipam-controller:latest: failed commit on ref "manifest-sha256:79e40e9a1578696bf89fa7b377ef8b430ae61f891f37203cd010db496c62fc25": unexpected status from PUT request to https://ghcr.io/v2/kubevirt/ipam-controller/manifests/sha256:79e40e9a1578696bf89fa7b377ef8b430ae61f891f37203cd010db496c62fc25: 403 Forbidden
Error: buildx failed with: ERROR: failed to solve: failed to push ghcr.io/kubevirt/ipam-controller:latest: failed commit on ref "manifest-sha256:79e40e9a1578696bf89fa7b377ef8b430ae61f891f37203cd010db496c62fc25": unexpected status from PUT request to https://ghcr.io/v2/kubevirt/ipam-controller/manifests/sha256:79e40e9a1578696bf89fa7b377ef8b430ae61f891f37203cd010db496c62fc25: 403 Forbidden
```